### ISSUE TITLE
[Suggestion] prevent a command to log in the console

### ISSUE THREAD

**Issue by noftaly** [SkriptLang/Skript#1477]
Hi, it will be super nice if you could add a way to prevent command logging in the console. This would be very useful for login skripts. The syntaxe will be something like this:
```command /mycommand:
    aliases: mc
    log: false
    trigger:
        #stuff```

**Comment by TheBentoBox** [comment]
Per-command setting for this makes a lot of sense!

**Comment by Pikachu920** [comment]
```
silent command /mycommand:
  trigger:
    # stuff
```
might be better, I don't know how I feel about a boolean entry

**Comment by TheBentoBox** [comment]
That looks good too. I think I prefer that since with recent additions like command cooldowns the entries section of a command can get rather long as is.

**Comment by noftaly** [comment]
I agree. And I think that putting it in the entries section is not such a bad thing, cause like this we can know everything about a command in a second, not looking everywhere if it is silent and stuff.
Also i’m not english so idk if this word even exist but maybe a `logless command /mycommand
(And sorry for my english ^^)
